### PR TITLE
Fix site build

### PR DESCRIPTION
**Pull request by slawekjaranowski**

### CODE REVIEW

File: doxia-doc-renderer/src/main/javadoc/org/apache/maven/doxia/docrenderer/itext/package.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
 -->
 
 <body>
-  <h1><b>DEPRECATED</b>.</h1>
+  <b>DEPRECATED</b>.
```
Comment (michael-o) [inline doxia-doc-renderer/src/main/javadoc/org/apache/maven/doxia/docrenderer/itext/package.html:23]: Drop this hunk, it has already been removed in an open PR.

**Comment by slawekjaranowski** [comment]
Can introduce conflict for: #21

**Review by michael-o — CHANGES_REQUESTED** [review]

File: doxia-doc-renderer/src/main/javadoc/org/apache/maven/doxia/docrenderer/itext/package.html (hunk 1 above)
Comment (slawekjaranowski) [reply doxia-doc-renderer/src/main/javadoc/org/apache/maven/doxia/docrenderer/itext/package.html:23]: I know, I can wait for merge your PR

**Comment by michael-o** [comment]
@slawekjaranowski Please rebase.

**Comment by slawekjaranowski** [comment]
@michael-o rebased, build pass

**Review by slachiewicz — APPROVED** [review]

**Pull request merged**